package.json

```diff
@@ -7,14 +7,14 @@
     "./src/crypto-sha1-2.js": "./src/crypto-sha1-2-browser.js"
   },
   "scripts": {
-    "test": "aegir-test",
-    "test:browser": "aegir-test browser",
+    "test": "aegir-test --webworker",
```

Comment:
Ah no, sorry this means run tests also in webworker, that's why it's not `--env webworker`.